src/applications/simple-forms/form-upload/containers/EditLink.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { VaLink } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+const EditLink = ({ href, router }) => {
+  const [link, setLink] = useState(null);
+
+  useEffect(
+    () => {
+      setLink(href);
+    },
+    [href],
+  );
+
+  function onClick(e) {
+    e.preventDefault();
+    router.push(link);
+  }
+
+  return <VaLink href={href} onClick={onClick} text="Edit" />;
+};
+
+function mapStateToProps(state) {
```

Comment:
can get rid of this block and the connect wrapper

Reply:
It looks like we do need this for the props (`router`) to get through.